### PR TITLE
Instructions update

### PR DESCRIPTION
**Pull request by TomasLenc**
Allowed for the participant to go back-forth in the instruction pages.

**tapTrainer**:
+ Instructions factored to decrease amount of text per page.
+ When tapping, instructions now stay in the middle of the screen.
+ Nicer visual feedback about tapping performance added.
+ Emphasized to keep tapping even if they cannot do it properly (don't
give up).

Also, leaving more space at the begining of track 2 without cue.

### CODE REVIEW

**Comment by CerenB** [comment]
I'm making minor changes in the instructions for small wordings. 
But I do not understand this sentence in `instrTrainingAfterSeq1-2`
Also in `instrTrainingBeforeSeq3-1`, could we find something to say "it would be last enriched musical sample" because as of now, I thought we reached the last part of the training. And/or in `instrTrainingIntro6`, we can say "we will play musical and simple rhythms to you.."

**Comment by CerenB** [comment]
A small visual question: what does that "v" sign doing in the visualisation of how well the participant is doing? The "#" changes according to the performance but "v" puzzled me a bit.

**Comment by TomasLenc** [comment]
> A small visual question: what does that "v" sign doing in the visualisation of how well the participant is doing? The "#" changes according to the performance but "v" puzzled me a bit.
It is the threshold you need to pass :) 
I wasn't sure if it would be intuitive, so thanks for the feedback. 
Maybe we could somehow mark what it means? No idea...

**Comment by CerenB** [comment]
Not sure if we need to mark it.. They should try doing their best :D 
If you want to try, maybe we can add a `|` on top of the line so it would be -|_________|___|+ 
Then we would need to add "pass"  on top of that I

**Comment by TomasLenc** [comment]
> But I do not understand this sentence in `instrTrainingAfterSeq1-2`

You mean this sentence? 
> Often we somehow feel the heartbeat of music pulsating, 
but there is no single sound that marks this pulse. 
We will explore this in the following example. 

How about something like this instead: 
> Often we somehow feel the heartbeat of music pulsating, 
even when there is no single instrument directly playing the pulse. 
We will explore this in the following example.

**Comment by CerenB** [comment]
OK. I do understand it but I had to read 2-3 times.

**Comment by TomasLenc** [comment]
> OK. I do understand it but I had to read 2-3 times.

Ok, one more attempt, and tell me which one you think is least confusing: 
> Sometimes, there is no instrument that directly plays the pulse.
But we can still somehow feel the heartbeat of music pulsating.

**Comment by CerenB** [comment]
Yes this is better! The last one, I find it more clear.

File: lib/sortNatural.m [hunk 1]
```diff
@@ -0,0 +1,95 @@
+function [cs,index] = sort_nat(c,mode)
```
Comment (CerenB) [inline lib/sortNatural.m:1]: the function is not named as the .m file

**Pull request merged**